lisa/runner.py

```diff
@@ -229,16 +229,14 @@ async def start(self) -> None:
 
         if self._results_collector:
             results = [x for x in self._results_collector.results.values()]
+            self.exit_code = 0
```

Comment:
It should be set in else after L239. If there there is any failure above, the code is -1.

Reply:
updated